### DISCUSSION TITLE
Launch Plus Server on Ubuntu

### DISCUSSION THREAD

**Discussion by Karl-Philippe** [PlusToolkit/PlusLib#910] [Q&A] [answered]
Hi,

I’m trying to launch Plus Toolkit in Ubuntu 20.04.3 LTS and I have succeded to build PLUS without errors. However, I don't see how to get an executable file to launch the Plus Server.

I have built PLUS using the following instructions https://github.com/PlusToolkit/PlusBuild/blob/master/Docs/BuildInstructionsLinux.md

What exacly do I need to build in the PlusBuild to be able to lauch the server? Or, could you tell me the steps to follow to launch the server on Ubuntu?

Thank you,

Karl-Philippe Beaudet

**Comment by adamrankin** [comment, marked as answer]
After building, you'll find the PlusServer binary in the bin directory.

You can launch it with many command line options, but the easiest is
`./PlusServer --config-file=XYZ.xml`

Alternatively, if you've built with Qt, you can use
`./PlusServerLauncher`
in the same folder for a GUI

**Reply by Karl-Philippe** [reply]
Thanks! The ./PlusServerLaucher command worked even tho I did not built with Qt.

**Comment by Telang8100** [comment]
hi,
I’m trying to launch Plus Toolkit in Ubuntu 20.04 and I have succeeded to build PLUS Build without errors.I'm using Realsense camera for image capture and display the live feed in 3D slicer.Used this link for build [https://github.com/PlusToolkit/PlusBuild/blob/master/Docs/BuildInstructionsLinux.md](url) .After build is complete.I run `./PlusServerLauncher` and loaded the `PlusDeviceSet_Server_IntelRealSenseVideo.xml` file for realsense depth camera.After loading file we were trying to connect to 3D slicer through OpenIGTLink but it was showing below error

![Error Image](https://user-images.githubusercontent.com/41721821/168794766-8f1f0541-a52f-4d0d-bbb9-5c0c4c01c1c6.png)

**Reply by adamrankin** [reply]
Could you paste the errors in the log (click the red circle) or upload the log file?

**Reply by Telang8100** [reply]
Hi 
Below is the log file for error
[051922_095832_PlusLog.txt](https://github.com/PlusToolkit/PlusLib/files/8723916/051922_095832_PlusLog.txt)

**Reply by adamrankin** [reply]
You must enable the RealSense in your build of Plus. Are you in the same group as #949 ?

**Reply by Telang8100** [reply]
Yes we are in same group but we are trying different things.
Can you please give me idea about how to enable the Realsense in Plus Build for ubuntu 20.04.If possible stepwise because I'm new to this things.

**Reply by adamrankin** [reply]
Yes, you'll have to build Plus with realsense enabled. In PlusBuild Cmake, enable the checkbox (if cmake-gui) or ON/OFF (if ccmake) for RealSense and rebuild PlusBuild. It will complain and ask you for the location of your realsense sdk. I found the easiest was to install librealsense2-dev and then give it the library location
